src/SailthruChannel.php

```diff
@@ -67,6 +68,23 @@ public function send(
                 static::getDefaultVars()
             );
 
+            if (config('services.sailthru.whitelist.enabled') === true) {
```

Comment:
Should this check affect brk?May be it should have been ideal to consider a tenant specific feature toggle instead?Just for consistency